routes/v0/cases.js

```diff
@@ -47,7 +21,7 @@ casesRouter.post('/', publicEndpoint, async (req, res) => {
   res.json(createdCase);
 });
 
-casesRouter.put('/:id', publicEndpoint, async (req, res) => {
+casesRouter.put('/:id', canEditCase, async (req, res) => {
```

Comment:
Editing cases is a tough first permission to add, because `PUT Case` is a big blob of stuff.  So we just use one middleware and have to push all the logic down into it.  But I imagine that for future endpoints, we can do more with our middleware.  For example, if editing a contact requires that it's either the counselor who owns the case, the supervisor of the office, or an admin, we could have:
`contactsRouter.put('/:id', isContactOwner, isOfficeSupervisor, isAdmin, async...`